javascript/javascript3/week3/shoppingCart/main.js

```diff
@@ -0,0 +1,114 @@
+class Product {
+    constructor(name, price) {
+        this.name = name;
+        this.price = price;
+    }
+    convertToCurrency(currency) {
+        if (currency.toLowerCase() === "dollar") {
+            const dollar = this.price * 0.14;
+            return dollar;
+        } else if (currency.toLowerCase() === "euro") {
+            const euro = this.price * 0.13;
+            return euro;
+        } else if (currency.toLowerCase() === "rupee") {
+            const rupee = this.price * 23.03;
+            return rupee;
+        }
+        return this.price;
+    }
+}
```

Comment:
it's better to use API for currency exchange rates not fixed numbers
but this is not the task in this exercise so just something to consider in the future